src/common/tusb_fifo.c

```diff
@@ -91,28 +93,117 @@ static inline uint16_t _ff_mod(uint16_t idx, uint16_t depth)
   return idx;
 }
 
+// Intended to be used to read from hardware USB FIFO in e.g. STM32 where all data is read from a constant address
+// Code adapted from dcd_synopsis.c
+static void _tu_fifo_read_from_const_src_ptr(void * __restrict dst, const void * __restrict src, uint16_t len)
```

Comment:
This code is specific to stm32, we shouldn't include it here. What is its pros, if the benefit is not that great. I would prefer to not include it.

Reply:
This is also valid for other MCUs. At the time of commenting it was done for STM32.

Reply:
For DA1469x register address is valid for 8 bits reading only not for 32 bits.

Reply:
Ah i see, thanks that was not covered!